habitat/config/default.py

```diff
@@ -226,6 +226,7 @@ def __init__(self, *args, **kwargs):
 SIMULATOR_SENSOR.HFOV = 90  # horizontal field of view in degrees
 SIMULATOR_SENSOR.POSITION = [0, 1.25, 0]
 SIMULATOR_SENSOR.ORIENTATION = [0.0, 0.0, 0.0]  # Euler's angles
+SIMULATOR_SENSOR.SENSOR_SUBTYPE = "PINHOLE"
```

Comment:
This is an enum on the sim side.  Is this getting remapped?

Reply:
Yeah, you can't represent enums in YAML files easily.